src/components/ProfileCard/index.js

```diff
@@ -26,10 +26,12 @@ const ProfileCard = ({ photoUrl, name, role, description, twitter, linkedin, git
           {role}
         </Text>
         <div className="flex flex-col justify-between flex-1">
-          <Text variant="base" className="flex-1">
-            {description}
-          </Text>
-          <div className="flex justify-center gap-[10%] justify-self-end lg:justify-start">
+          {isDescription ? null :
```

Comment:
If you make that `isDescription` is `true` when there is a description (see the comment above), this can be simplified to 

```
{isDescriotion & <Text ...>...</Text>}
```

Reply:
@varya thank you